File: exercicio_1008.java
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        double C = sc.nextDouble();
        
        double SALARY = (C*B);
        
        System.out.printf("NUMBER = %d\n",A);
        
        System.out.printf("SALARY = U$ %.2f\n", SALARY);
        
    }
}

File: exercicio_1008.c
#include <stdio.h>
 
int main() {
 int num, hr;
 float sl, total;
 
 scanf("%d", &num);
 scanf("%d", &hr);
 scanf("%f", &sl);
 
 total = sl*hr;

printf("NUMBER = %d\n", num);
printf("SALARY = U$ %.2f\n",total);
    return 0;
}
